src/SmartComponents/Recs/_Details.scss

```diff
@@ -1,4 +1,4 @@
-@import '~@redhat-cloud-services/frontend-components-inventory-insights/esm/Insights.css';
+@import '~@redhat-cloud-services/frontend-components-inventory-insights/Insights/Insights.scss';
```

Comment:
This failed with `Insights.scss` for me and the file on my systems was `insights.scss`

Reply:
did yah blow away n reinstall  yer node modules? they redid those packages... where things are (in them) changed
🤔

Reply:
> did yah blow away n reinstall yer node modules? they redid those packages... where things are (in them) changed

I didn't.  Just tried it and it worked :facepalm: